### PR TITLE
Allow variable stabilization parameters for each field

### PR DESCRIPTION
**Pull request by gassmoeller**
This change allows to prescribe different values for the stabilization parameters `beta` and `cR` for the temperature and each compositional field. This might be useful if for example smaller values are possible for the temperature field than the compositional fields, because the temperature field contains some physical diffusion. The test only shows that different values are possible, and are correctly used inside the code, it does not claim that these values are stable. I need to write a description for the test tomorrow, but the rest should be more or less ready.

### CODE REVIEW

File: source/simulator/parameters.cc [hunk 1]
```diff
@@ -848,7 +848,7 @@ namespace aspect
                            "thermal expansion coefficient, also commonly referred to as $\\alpha$."
                            "Units: None.");
         prm.declare_entry ("cR", "0.33",
-                           Patterns::Double (0),
+                           Patterns::List(Patterns::Double (0)),
                            "The $c_R$ factor in the entropy viscosity "
                            "stabilization. This parameter controls the part of the entropy viscosity "
```
Comment (tjhei) [inline source/simulator/parameters.cc]: can you update the description?

File: tests/artificial_viscosity_list/log.txt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+--     . using deal.II 9.0.0
+--     . using Trilinos 12.10.1
+--     . using p4est 2.0.0
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+-----------------------------------------------------------------------------
+
+-----------------------------------------------------------------------------
+-- For information on how to cite ASPECT, see:
+--   https://aspect.geodynamics.org/citing.html?ver=2.1.0-pre&sha=7345fcaf4&src=code
```
Comment (tjhei) [inline tests/artificial_viscosity_list/log.txt]: this is cool to see here. ;-)

File: tests/artificial_viscosity_list/log.txt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+   Solving Stokes system... 110+0 iterations.
+
+   Postprocessing:
+     Writing graphical output:           output-shell_simple_3d/solution/solution-00002
+     RMS, max velocity:                  0.00117 m/year, 0.00275 m/year
+     Temperature min/avg/max:            973 K, 1478 K, 1973 K
+     Heat fluxes through boundary parts: -1.888e+12 W, 5.815e+12 W
+     Writing depth average:              output-shell_simple_3d/depth_average.txt
+
+*** Timestep 3:  t=2.55587e+08 years
```
Comment (tjhei) [inline tests/artificial_viscosity_list/log.txt]: cut off?

**Comment by gassmoeller** [comment]
Done and tester is happy. Ready from my side.

**Pull request merged**